tests/scripts/all.sh

```diff
@@ -505,6 +505,30 @@ tests/ssl-opt.sh -f RSA
 msg "test: RSA_NO_CRT - RSA-related part of compat.sh (ASan build)" # ~ 3 min
 tests/compat.sh -t RSA
 
+# Re-run with reduced output message buffer
+msg "build: with ASan, small output buffer"
```

Comment:
Please put the most important part first and make the message more informative:
```
msg "build: small SSL_OUT_CONTENT_LEN (ASan build)"
msg "test: small SSL_OUT_CONTENT_LEN - ssl-opt.sh MFL-related and large packet tests"
```
(I'd rather drop the `(Asan build)` part of the test description and keep within 79 columns.)